src/app/hearing-part/components/listing-note-list/listing-note-list.component.ts

```diff
@@ -1,73 +1,68 @@
-import * as moment from 'moment';
 
 @Component({
-  selector: 'app-listing-note-list',
-  templateUrl: './listing-note-list.component.html',
-  styleUrls: ['./listing-note-list.component.scss']
+    selector: 'app-listing-note-list',
+    templateUrl: './listing-note-list.component.html',
+    styleUrls: ['./listing-note-list.component.scss']
 })
 export class ListingNoteListComponent {
-  @ViewChild('specialNotesList') specialNotesList: NoteListComponent;
-  @ViewChild('newNoteList') newNoteList: NoteListComponent;
+    @ViewChild('specialNotesList') specialNotesList: NoteListComponent;
+    @ViewChild('newNoteList') newNoteList: NoteListComponent;
 
-  public specialNoteViewModels: NoteViewmodel[] = [];
-  public newNoteViewModels: NoteViewmodel[] = [this.listingNotesConfig.noteViewModelOf(NoteType.OTHER_NOTE)];
-  public oldNoteViewModels: NoteViewmodel[] = [];
+    public specialNoteViewModels: NoteViewmodel[] = [];
+    public newNoteViewModels: NoteViewmodel[] = [this.listingNotesConfig.noteViewModelOf(NoteType.OTHER_NOTE)];
+    public oldNoteViewModels: NoteViewmodel[] = [];
 
-  @Input() public entityId: string;
-  @Input() set notes(notes: Note[]) {
-    this.specialNoteViewModels = [];
-    this.oldNoteViewModels = [];
-    this.newNoteViewModels  = [this.listingNotesConfig.noteViewModelOf(NoteType.OTHER_NOTE)];
+    @Input() public entityId: string;
 
-    notes
-        .map(getNoteViewModel)
-        .forEach(this.disposeToProperArrays);
+    @Input() set notes(notes: Note[]) {
+        this.specialNoteViewModels = [];
+        this.oldNoteViewModels = [];
+        this.newNoteViewModels = [this.listingNotesConfig.noteViewModelOf(NoteType.OTHER_NOTE)];
 
-    this.specialNoteViewModels = [
-        this.listingNotesConfig.getOrCreateNote(this.specialNoteViewModels, NoteType.SPECIAL_REQUIREMENTS),
-        this.listingNotesConfig.getOrCreateNote(this.specialNoteViewModels, NoteType.FACILITY_REQUIREMENTS),
-    ];
+        notes
+            .map(getNoteViewModel)
+            .forEach(this.disposeToProperArrays);
 
-    this.oldNoteViewModels = this.sort(this.oldNoteViewModels);
-  }
+        this.specialNoteViewModels = [
+            this.listingNotesConfig.getOrCreateNote(this.specialNoteViewModels, NoteType.SPECIAL_REQUIREMENTS),
+            this.listingNotesConfig.getOrCreateNote(this.specialNoteViewModels, NoteType.FACILITY_REQUIREMENTS),
+        ];
 
-  constructor(readonly listingNotesConfig: ListingCreateNotesConfiguration,
-            public notePreparerService: NotesPreparerService) { }
+        this.oldNoteViewModels = sortNotesByLatestFirst(this.oldNoteViewModels);
+    }
 
-  prepareNotes() {
-    const preparedFreeTextNotes = this.notePreparerService.prepare(
-        this.specialNotesList.getModifiedNotes(),
-        this.entityId,
-        this.listingNotesConfig.entityName
-    );
+    constructor(readonly listingNotesConfig: ListingCreateNotesConfiguration,
```

Comment:
is `listingNotesConfig` public by default? If so - why do we define `public` for the other argument?

I'd rather use private/public here even if redundant (but I'm not going to argue about it - personal preference).

Reply:
good question, I would even follow it, why both of them are public, if not used from html.
I can change it to private as it should be.